tools/astarte_export/lib/LogFmtFormatter.ex

```diff
@@ -0,0 +1,102 @@
+#
```

Comment:
Do you mind to remove this file and to switch to pretty log library?

Reply:
Updated with pretty log, removed LogFmtFormatter.ex